private-templates-service/client/src/components/Dashboard/RecentlyViewed/RecentlyViewedTable.tsx

```diff
@@ -50,14 +49,8 @@ class RecentlyViewedTable extends React.Component<Props> {
           </RecentlyViewedItem>
           <RecentlyViewedItem>
             <TemplateStateWrapper>
-              <RecentlyViewedStatusIndicator
-                state={
-                  template.isLive
-                    ? PostedTemplate.StateEnum.Live
-                    : PostedTemplate.StateEnum.Draft
-                }
-              />
-              <Status>{template.isLive ? "Published" : "Draft"}</Status>
+              <RecentlyViewedStatusIndicator state={template.instances[0].state}/>
+              <Status>{template.instances[0].state && template.instances[0].state.toString().charAt(0).toUpperCase() + template.instances[0].state.toString().slice(1)}</Status>
```

Comment:
Can you extract this logic out to a const to make it easier to understand whats going on?

Reply:
Moved to utils file, thanks!